apps/admin/src/modules/configurations/components/ConfigurationForm.vue

```diff
@@ -1,14 +1,15 @@
 <template>
 	<el-form @submit.native.prevent="save">
-    <template v-for="key in keys" :key="key">
-      <el-switch
+    <template v-for="key in keys">
+      <el-switch :key="key + '-switch'"
```

Comment:
is :key an input? (been a while since I used vue) 
+ what if options is empty/null?  `v-if="valuesTypes[key].options.length === 2"`

Reply:
I agree, I think it should be something like:
v-if="valuesTypes[key].options && valuesTypes[key].options.length === 2"

Reply:
Or maybe with optional chaining

Reply:
The key is for the v-for loop